typescript/libs/global_vars.py

```diff
@@ -2,13 +2,14 @@
 import re
 
 import sublime
+from os.path import dirname
 from .logger import *
```

Comment:
I mean ``logging`` as the global python module, I wasn't referring to the ``logger`` file in this plugin. That was the only thing used in the ``global_vars.py`` anyways.